### DISCUSSION TITLE
min browser and findTo?

### DISCUSSION THREAD

**Discussion by ghost** [minbrowser/min#2137] [Q&A] [answered]
Hi everyone.

I would like to know what you all think of the min browser with [findTo](https://github.com/lucasm/findto). *What do you all think about adding findTo main page in Min browser?* Can anybody help me?

#### question/feedback
What do you all think of the idea?

**Comment by PalmerAL** [comment, marked as answer]
Rather than having a specific integration for this, we should probably have an option in settings to set a new tab page URL, and you can set any service you want.

**Reply by ghost** [reply]
Hi PalmerAL.

> Rather than having a specific integration for this, we should probably have an option in settings to set a new tab page URL, and you can set any service you want.

thank you for feedback.
